python/utilities/log_list/java_generator.py

```diff
@@ -0,0 +1,73 @@
+import datetime
+import textwrap
+import base64
+
+def _write_java_header(f, package):
+    year = datetime.date.today().year
+    f.write(
+        "/*\n"
+        " * Copyright (C) %(year)d The Android Open Source Project\n"
+        " *\n"
+        " * Licensed under the Apache License, Version 2.0 (the \"License\");\n"
+        " * you may not use this file except in compliance with the License.\n"
+        " * You may obtain a copy of the License at\n"
+        " *\n"
+        " *      http://www.apache.org/licenses/LICENSE-2.0\n"
+        " *\n"
+        " * Unless required by applicable law or agreed to in writing, software\n"
+        " * distributed under the License is distributed on an \"AS IS\" BASIS,\n"
+        " * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n"
+        " * See the License for the specific language governing permissions and\n"
+        " * limitations under the License.\n"
+        " */\n\n"
+        "/* This file is generated by print_log_list.py\n"
+        " * https://github.com/google/certificate-transparency/blob/master/python/utilities/log_list/print_log_list.py */\n\n"
+        "package %(package)s;\n\n" %
+        {"year": year,
+         "package": package})
+
+def _encode_key(description, key):
+    unsigned = (ord(c) for c in key)
+    signed = (u - 256 if u > 127 else u for u in unsigned)
+    return ("        // " + description + "\n"
+            "        new byte[] {\n"
+            + textwrap.fill(", ".join("%d" % s for s in signed), width=85,
```

Comment:
Could you assign the result from this call to a variable you'll include in the string generated here, so the code is slightly more readable?